docker-compose.yaml

```diff
@@ -19,6 +19,6 @@ services:
       - "/etc/localtime:/etc/localtime:ro"
       - "/etc/timezone:/etc/timezone:ro"
       - "./softether_data:/mnt"
-      - "./softether_log:/root/server_log"
-      - "./softether_packetlog:/root/packet_log"
-      - "./softether_securitylog:/root/security_log"
+      - "./softether_log:/usr/local/bin/server_log"
+      - "./softether_packetlog:/usr/local/bin/packet_log"
+      - "./softether_securitylog:/usr/local/bin/security_log"
```

Comment:
are those log folders good to keep in /usr/loca/bin ?

Reply:
As this is the default of vpnserver i just did not want to mess around with it but for sure they should go to /var/log.

Reply:
ok

Reply:
Is there any way to alter this right on startup or would it make sense to built this right into vpnserver?

Reply:
it might be changed during build

```
These directories can be changed at compile-time by specifying via CMake variables.
* `SE_PIDDIR` - PID directory
* `SE_LOGDIR` - root log directory
* `SE_DBDIR`  - config files and variable state directory

```